README.md

```diff
@@ -1,5 +1,8 @@
 # IOpipe Analytics & Distributed Tracing Agent for Python
 
+[![PyPI version](https://badge.fury.io/py/iopipe.svg)](https://badge.fury.io/py/iopipe)
```

Comment:
Yay badges! This is great

Reply:
@pselle I assume we want both slack links to go to iopipe.now.sh?

Reply:
Ignore previous comment. Yes, both to now.sh.